GliaWidgets/Public/Glia/Glia.Deprecated.swift

```diff
@@ -84,4 +84,24 @@ extension Glia {
             sceneProvider: sceneProvider
         )
     }
+
+    /// Deprecated, use `clearVisitorSessionIfPossible() throws` instead.
+    @available(
+        *,
+         deprecated,
+         message: "Use clearVisitorSessionIfPossible() throws instead."
+    )
+    public func clearVisitorSession() {
+        do {
+            try clearVisitorSessionIfPossible()
+        } catch {
+            print("⚠️ `clearVisitorSession` failed with error: `\(error)`")
+        }
+    }
+
+    /// Deprecated, use ``callVisualizer.showVisitorCodeViewController`` instead.
+    @available(*, deprecated, message: "Deprecated, use ``CallVisualizer.showVisitorCodeViewController`` instead.")
+    public func requestVisitorCode(completion: @escaping (Result<VisitorCode, Swift.Error>) -> Void) {
```

Comment:
I don't remember, was this method part of the SDK before the call visualizer project? If it was, then it's fine that it's deprecated. If it was done during the CV project, then it doesn't need to be deprecated because it was never officially released.

Reply:
Good point. I'll double check it

Reply:
We added this interface in January. Then we had 3 releases. Form one side, that were not releases for CV, but potentially some integrators could use it. And this deprecation was made in February and I just moved it to appropriate file. 
Based on that I would leave it as deprecated. 
@igorkravchenko @rasmustautsglia @yurii-glia wdyt?

Reply:
If it was not a part of new call visualizer, but officially (mentioned in documentation) released prior to that, it makes sense to deprecate it.

Reply:
But this interface deprecation was already published also.

Reply:
Ahhh okay, then leave it as it is.